addons/medical_gui/functions/fnc_canOpenMenu.sqf

```diff
@@ -18,8 +18,13 @@
 
 params ["_player", "_target"];
 
-alive _player
-&& {!IS_UNCONSCIOUS(_player)}
-&& {!isNull _target}
-&& {_player distance _target < GVAR(maxDistance) || {vehicle _player == vehicle _target}}
-&& {GVAR(enableMedicalMenu) == 1 || {GVAR(enableMedicalMenu) == 2 && {vehicle _player != _player || {vehicle _target != _target}}}}
+// If in Zeus
+if (!isNull findDisplay 312) exitWith {
+    !isNull _target &&
```

Comment:
```suggestion
    !isNull _target && {EGVAR(medical_treatment,treatmentTimeCoeffZeus) != 10} &&
```
not sure, it might be nice to have a way to disable
and I don't want to add a brand new setting
this sorta works (disable if time set to max) but is a little ugly

Reply:
timeCoef of zero would make more sense to me

Reply:
> not sure, it might be nice to have a way to disable and I don't want to add a brand new setting this sorta works (disable if time set to max) but is a little ugly

@PabstMirror What do you mean by "disabled"? That the setting doesn't have an effect or that you can't open the medical menu via zeus at all?

If it's the latter, then I suggest extending the CBA setting slider from `[0, 10]` to `[-1, 10]` and if the coeff is below 0, it would be considered to be turned off. Personally I would like to have the ability to set the time to 0.
But I don't understand why you would want to be able to enable/disable zeus medical menu access though.

However, if it's the prior, setting the coeff to `1` means no change.

Reply:
overloading this setting was a bad idea and is just going to confuse everyone
and most people won't care if zeus heals
we could just add a simple getVar "setting" so it could be disabled if a mission maker/group desires

```suggestion
    !isNull _target && {missionNamespace getVariable [QGVAR(enableZeusModule), true]} &&
```